cadquery/occ_impl/exporters/__init__.py

```diff
@@ -106,7 +106,12 @@ def export(
             shape.exportStep(fname)
 
     elif exportType == ExportTypes.STL:
-        shape.exportStl(fname, tolerance, angularTolerance)
+        if hasascii := [
+            opt.get(k) for k in ["ascii", "ASCII"] if opt is not None and k in opt
+        ]:
+            shape.exportStl(fname, tolerance, angularTolerance, hasascii.pop())
+        else:
+            shape.exportStl(fname, tolerance, angularTolerance)
```

Comment:
Maybe this is easier to follow?

```suggestion
        if opt:
            useascii = opt.get("ascii", False) or opt.get("ASCII", False)
        else:
            useascii = False
        
        shape.exportStl(fname, tolerance, angularTolerance, useascii)
```

Reply:
Agreed.  Thanks @adam-urbanczyk !